packages/react-components/src/components/MessageThread.tsx

```diff
@@ -359,9 +361,9 @@ const memoizeAllMessages = memoizeFnAll(
     switch (message.messageType) {
       case 'chat': {
         const myChatMessageStyle =
-          styles?.myChatMessageContainer || message.status === 'failed'
-            ? FailedMyChatMessageContainer
-            : defaultMyChatMessageContainer;
+          message.status === 'failed'
+            ? styles?.failedMyChatMessageContainer || FailedMyChatMessageContainer
```

Comment:
In the event `failedMyChatMessageContainer` this should fallback to `myChatMessageContainer` (i.e. we consider failedMyChatMessageContainer to be a more specific override).

```suggestion
            ? styles?.failedMyChatMessageContainer ?? styles?.myChatMessageContainer ?? FailedMyChatMessageContainer
```

Reply:
Updated according to your suggestion 👍